packages/geo/src/lib/feature/shared/strategies/in-map-extent.ts

```diff
@@ -0,0 +1,111 @@
+import { unByKey } from 'ol/Observable';
+import * as olextent from 'ol/extent';
+
+import { EntityStoreStrategy } from '@igo2/common';
+
+import { FeatureStore } from '../store';
+import { FeatureStoreInMapExtentStrategyOptions, Feature } from '../feature.interfaces';
+import { Subscription } from 'rxjs';
+
+/**
+ * This strategy maintain the store features updated while the map is moved.
+ * The features's state inside the map are tagged inMapExtent = true;
+ */
+export class FeatureStoreInMapExtentStrategy extends EntityStoreStrategy {
+
+  /**
+   * Subscription to the store's OL source changes
+   */
+  private stores$$ = new Map<FeatureStore, string>();
```

Comment:
Is this used? I don't see where values are set in it.

Reply:
As in loading-layer.ts

https://github.com/infra-geo-ouverte/igo2-lib/blob/next/packages/geo/src/lib/feature/shared/strategies/loading-layer.ts#L22

Reply:
You're right. The loading-layer nevers updates the stores$$ map either. It should contain the key to each OL listener. It could become a performance issue.